uax/uax42/Readme.md

```diff
@@ -0,0 +1,24 @@
+# Generating TR42
+
+## Step 1 - Generate property value fragments
+
+- Run org.unicode.xml.GeneratePropertyValues to populate the UNICODETOOLS_REPO_DIR/uax/uax42/fragments/ folder.
+
+## Step 2 - Generate TR42 index.html and index.rnc 
+
+- In UNICODETOOLS_REPO_DIR/uax/uax42/ run `mvn xml:transform`
+
+  index.html and index.rnc will be generated in UNICODETOOLS_REPO_DIR/uax/uax42/output/
+
+## Step 3 - Validate generated UAX XML files
+
+You'll need a [RELAX NG](https://relaxng.org/) schema validator. We'll use [jing-trang](https://github.
+com/relaxng/jing-trang) in this example.
+
+1. Clone and build [jing-trang](https://github.com/relaxng/jing-trang)
```

Comment:
This does not look workable. Dependencies should be handled by Maven via the pom.xml file which I assume would pull the Trang artifacts from https://mvnrepository.com/artifact/org.relaxng/trang

It should be possible to run one Maven command that builds and runs the tool.